### PR TITLE
Set mnemonic widgets and accessible labels

### PR DESCRIPTION
**Pull request by danirabbit**
This makes sure that the screen reader can read out the proper labels for widgets as they are navigated to. While here, also make sure that we're setting focus on the next list in the variant view. Previously you'd hit enter and then the screen reader would read out the back button label instead of the next option

### CODE REVIEW

File: src/Views/AccountView.vala [hunk 1]
```diff
@@ -62,10 +64,14 @@ public class Installer.AccountView : AbstractInstallerView {
             input_purpose = NAME
         };
 
+        ((Gtk.Label) realname_label.get_first_child ()).mnemonic_widget = realname_entry;
```
Comment (tintou) [inline src/Views/AccountView.vala:67]: I believe that it should be something like:
```vala
realname_entry.update_relation (Gtk.AccessibleRelation.LABELLED_BY, realname_label);
```
I haven't had the time to test it though

Comment (danirabbit) [reply src/Views/AccountView.vala:67]: So the docs say:
> setting the mnemonic widget on a GtkLabel will update the GTK_ACCESSIBLE_RELATION_LABELLED_BY relation on the widget with a reference to the label

Which I read as being a recommendation

Comment (tintou) [reply src/Views/AccountView.vala:67]: It is just that I don't like the use of `realname_label.get_first_child ()` we should do something in granite for this

Comment (danirabbit) [reply src/Views/AccountView.vala:67]: Yeah I filed https://github.com/elementary/granite/issues/662. I'll make a branch today

**Review by lenemter — APPROVED** [review]

**Pull request merged**